dotCMS/src/main/java/com/dotcms/rendering/velocity/servlet/VelocityNavigateEditMode.java

```diff
@@ -79,6 +87,6 @@ public void serve(final OutputStream out) throws DotDataException, IOException,
     private PageMode getMode() {
         final PageMode currentMode = PageMode.get(request);
         return currentMode.showLive ? currentMode :
-                APILocator.getHTMLPageAssetRenderedAPI().getDefaultEditPageMode(user, request, uri);
+                APILocator.getHTMLPageAssetRenderedAPI().getDefaultEditPageMode(user, request, uri, response);
```

Comment:
why are you removing the response, some of the thing we have figured out is that the request must be always coming together with the response, even if it is not used at this time, could be in the future

Reply:
I am not removing the response